tools/workspace/new_release.py

```diff
@@ -77,6 +77,16 @@
     "sdformat_internal": "",
 }
 
+# Packages in these cohorts should be upgraded together (in a single commit).
+_COHORTS = (
+    # The uwebsockets depends on usockets;
+    # be sure to keep them in alignment.
+    ("uwebsockets", "usockets"),
+    # The sdformat depends on both gz libraries;
+    # be sure to keep them in alignment.
+    ("sdformat_internal", "gz_math_internal", "gz_utils_internal"),
```

Comment:
These should probably be sets. Also, the comments' grammar is poor.
```suggestion
    # uwebsockets depends on usockets; be sure to keep them aligned.
    {"uwebsockets", "usockets"},
    # sdformat depends on both gz libraries; be sure to keep them aligned.
    {"sdformat_internal", "gz_math_internal", "gz_utils_internal"},
```